src/XR/features/WebXRLayers.ts

```diff
@@ -219,10 +225,15 @@ export class WebXRLayers extends WebXRAbstractFeature {
             return false;
         }
 
-        this._glContext = this._xrSessionManager.scene.getEngine()._gl;
+        const engine = this._xrSessionManager.scene.getEngine();
+        this._glContext = engine._gl;
         this._xrWebGLBinding = new XRWebGLBinding(this._xrSessionManager.session, this._glContext);
         this._existingLayers = [];
-        this.addXRSessionLayer(this.createProjectionLayer());
+
+        const projectionLayerParams = engine.getCaps().multiview
+            ? defaultMultiviewXRProjectionLayerInit
+            : defaultXRProjectionLayerInit;
+        this.addXRSessionLayer(this.createProjectionLayer(projectionLayerParams));
```

Comment:
Perhaps it would be better to have the user specify `defaultMultiview` as a flag in the feature options since using multiview by default could potentially break simple (non-multiview) shaders right out of the box.

Reply:
Added a flag to indicate preference, defaulting to no multiview. Let me know what you think

Reply:
I also added an optional boolean to  `createProjectionLayer` to make adding multiview to the projection layer more explicit, and allows you to not use multiview if your projection layer is using 'texture-array' as the textureType, although I've been seeing some weird rendering bugs with non-multiview with 'texture-array' that we will need to address. I will look into it some more